habitat-lab/habitat/tasks/rearrange/rearrange_task.py

```diff
@@ -314,8 +314,8 @@ def _is_violating_safe_drop(self, action_args):
         )
         return (
             self._sim.grasp_mgr.is_grasped
-            and action_args.get("grip_action", None) is not None
-            and action_args["grip_action"] < 0
+            and action_args.get("grip_pick_action", None) is not None
+            and action_args["grip_pick_action"] < 0
```

Comment:
The naming is a bit weird to me. Can you change the "grip_pick_action" and "grip_place_action" to "grip_action" and "place_action" globally?

Reply:
OK